README.md

```diff
@@ -13,18 +13,17 @@
 ## Installation
 On a terminal:
 ```bash
-conda create -n qsui python=3.9
-conda activate qsui
-
-git clone https://github.com/aps-bcda/qsui
+git clone git@github.com:BCDA-APS/qsui.git
 cd qsui
-pip install .
+conda env create --force -n qsui -f ./env.yml
```

Comment:
Soon (but now for APS use), the use of `--force` will need to switch to `-y`.  When the version of conda is updated to the most recent one.

Reply:
- #4